bcda/models/models.go

```diff
@@ -528,9 +528,14 @@ func GetBlueButtonID(bb client.APIClient, modelIdentifier, patientIdMode, reqTyp
 	var foundIdentifier = false
 	var foundBlueButtonID = false
 	blueButtonID = patient.Entry[0].Resource.ID
+	// once this is set to true, what happens with future iterations??
```

Comment:
What does this comment mean?

Reply:
nothing false alarm, was over engineering in my head

Reply:
Can this comment be removed?

Reply:
done